modules/monitoring-reviewing-monitoring-dashboards-admin.adoc

```diff
@@ -19,6 +19,8 @@ In the *Administrator* perspective, you can view dashboards relating to core {pr
 
 . Optional: Select a time range for the graphs in the *Time Range* list.
 
+. Optional: Select a custom time range for the graphs by selecting *Custom time range* in the *Time Range* dropdown. Select the *From* and *To* dates and input the times for the custom range.
```

Comment:
This step calls it the *Time Range* dropdown. Previous step uses *Time Range* list.